### ISSUE TITLE
new-theme fails with invalid argument exception

### ISSUE THREAD

**Issue by bhattisatish** [getgrav/grav-plugin-devtools#14]
I just tried creating a new theme as per the https://learn.getgrav.org/themes/theme-tutorial

After entering all the information when prompted
I get the following exception

     [InvalidArgumentException]                 
     Identifier "flex-entries" is not defined.

This occurs irrespective of choosing "pure-blank" or "inheritence" option

**Comment by Vivalldi** [comment]
Do you have the full output? "flex-entries" is not caller nor defined anywhere in this project.

**Comment by bhattisatish** [comment]
If you try running the command
```
bin/plugin devtools new-theme
Enter Theme Name: anyname
Enter Theme Description: For testing purpose only
Enter Developer Name: yaday
Enter Developer Email: yada@example.com
Please choose a template type
  [0] pure-blank
  [1] inheritence
 > 1
Please choose a theme to extend: 
  [0] antimatter
  [1] g5_hydrogen

 > 0
                                             
  [InvalidArgumentException]                 
  Identifier "flex-entries" is not defined.  
                                            
new-theme [-pn|--name [NAME]] [-d|--description [DESCRIPTION]] [-dv|--developer [DEVELOPER]] [-e|--email [EMAIL]]

```

**Comment by flaviocopes** [comment]
Cannot replicate. I don't know where the `flex-entries` value comes from, as I cannot find any reference to it in the plugin or in Grav. 

Can you try if you can replicate on a clean Grav install, as some other plugin you have installed might interfere?

**Comment by rhukster** [comment]
Flex entries is from the Flex-Directory plugin.  Not sure why/how it's interfering, but i'll look into it.

**Comment by voznik** [comment]
I can confirm this issue still exists if `grav-plugin-flex-directory` is installed.
@rhukster can you pay attention to this?
I've also created [issue](https://github.com/trilbymedia/grav-plugin-flex-directory/issues/16) where this plugin prevents gantry5 functionality by intercepting POST requests

**Comment by voznik** [comment]
@rhukster I figured it out: somehow you forgot to merge this fix trilbymedia/grav-plugin-flex-directory#c382a750d0ee24702182650ec0fe135a09eab04c to `develop` branch

**Comment by rhukster** [comment]
Ugg.. just saw this, yes i put the fix in master not develop, fixed and released.
